test/components/NavBar/__snapshots__/OidcLoginComponent.test.jsx.snap

```diff
@@ -82,6 +83,7 @@ exports[`sets OIDC provider on login 1`] = `
   />
   <div>
     oidc.provider.url
+    http://localhost:3000
```

Comment:
We now pass the redirect URL in explicitly, causing it to render correctly in the test (it wasn't before, and I didn't notice).